flask_restless/helpers.py

```diff
@@ -180,10 +180,33 @@ def is_date_field(model, fieldname):
     """Returns ``True`` if and only if the field of `model` with the specified
     name corresponds to either a :class:`datetime.date` object or a
     :class:`datetime.datetime` object.
-
     """
-    fieldtype = get_field_type(model, fieldname)
-    return isinstance(fieldtype, Date) or isinstance(fieldtype, DateTime)
+    is_datelike = lambda field_type: isinstance(field_type, Date) or \
+        isinstance(field_type, DateTime)
+    return iteratively_check_parent_is_of_field(
```

Comment:
Don't place one argument per line; follow the first example under [argument indentation in PEP8][1], but only breaking the line when an argument would put the line length past 79 characters.

[1]: https://www.python.org/dev/peps/pep-0008/#indentation